package.json

```diff
@@ -84,6 +87,7 @@
     "node": ">=8"
   },
   "pre-commit": [
+    "update:yarn.lock",
```

Comment:
We now can check if the `yarn.lock` file is up-to-date and staged before the commit is allowed. Thus ensuring `yarn.lock` remains up-to-date.

Reply:
Alternatively we could use https://www.npmjs.com/package/check-yarn-lock and leave the updating to the dev?

Reply:
That package is for projects that use `yarn` but this one, currently, uses `npm` (it's on my list to change that long term...as is making this a published package instead of the commit SHA install we currently use).